pyramid_learning_journal/pyramid_learning_journal/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""Instantiate the inclusion."""
+from pyramid.config import Configurator
+import os
+
+
+
+def main(global_config, **settings):
+    """Function returns a Pyramid WSGI application."""
+    settings['sqlalchemy.url'] = os.environ.get('DATABASE_URL')
+    config = Configurator(settings=settings)
+    config.include('pyramid_jinja2')
+    config.include('.routes')
+    config.include('.models')
+    config.include('.security')
+    # config.include('.views')
+    config.add_static_view(name='static', path='pyramid_learning_journal:static')
```

Comment:
Any static view that you seek to add should be in your `routes.py` file with your other static views and routes